debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -633,6 +633,11 @@ public static void j9bcutil_dumpRomClass(PrintStream out, J9ROMClassPointer romC
 			}
 		}
 
+		if (J9ROMClassHelper.hasImplicitCreationAttribute(romClass)) {
+			U32 implicitCreationFlags = OptInfo.getImplicitCreationFlags(romClass);
+			out.format("ImplicitCreation flags: 0x%s%n", implicitCreationFlags.getHexValue());
```

Comment:
Please remove the redundant `0x` here (see [Scalar.getHexValue()](https://github.com/eclipse-openj9/openj9/blob/master/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/types/Scalar.java#L71)).

Reply:
updated